### ISSUE TITLE
Update query syntax

### ISSUE THREAD

**Issue by jl-wynen** [SciCatProject/scitacean#253]
Instead of the current 
```python
{'fields': '{"proposalId":"..."}', 'limits': '{"order": "creationTime:desc", "limit": 3}'}
```
it should be
```python
{ "where" : {"proposalId" : "..."},"limits": {"limit": 3, "order": "creationTime:desc"}}
```
The difference is `'fields'` -> `'where'`.

**Comment by nitrosx** [comment]
Supposedly, `fields` allows you to select which fields are returned for each dataset.

**Comment by jl-wynen** [comment]
This is incorrect. The old query still works. The problems I encountered were that I used incorrect field names. Tests fixed in #243
